amxmodx/srvcmd.cpp

```diff
@@ -63,6 +63,40 @@ void amx_command()
 
 		print_srvconsole("%d plugins, %d running\n", plugins, running);
 	}
+	else if (!strcmp(cmd, "plugin"))
+	{
+		auto id = atoi(CMD_ARGV(2));
+
+		if (CMD_ARGC() < 3)
+		{
+			print_srvconsole("Usage: amxx plugin < plugin_id >\nFor a list of plugins, use the \"amxx plugins\" command\n");
+		}
+		else
+		{
+			auto plugin = g_plugins.findPlugin(id);
+
+			if (plugin && plugin->isValid())
+			{
+				print_srvconsole("   Name: %s\n", plugin->getTitle());
+				print_srvconsole("   Version: %s\n", plugin->getVersion());
+				print_srvconsole("   Author: %s\n", plugin->getAuthor());
+				print_srvconsole("   URL: %s\n", plugin->getUrl());
+
+				auto description = plugin->getDescription(); 
+				if (description[0] != '\0')
```

Comment:
If AMXX is compiled in C++17, you can use

`if(auto blabla = ...; blablabla[0])`

Reply:
Judging by the error, all the same AMX Mod X is not compiled in C ++17. I revert changes.